pkg/build/graph/analysis/bc.go

```diff
@@ -162,6 +162,62 @@ func FindCircularBuilds(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 	return markers
 }
 
+// findPendingTags is the guts behind FindPendingTags .... break out some of the content and reduce some indentation
+func findPendingTags(uncastIstNode graph.Node, g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+	istNode := uncastIstNode.(*imagegraph.ImageStreamTagNode)
+	if istNode.Found() {
```

Comment:
> See now I feel bad because I suggested the opposite ... look for yourself #9308 (comment)

Eh, mostly right.  If this moves into the parent, the sig stays `istNode *imagegraph.ImageStreamTagNode, g osgraph.Graph, f osgraph.Namer` right?